quodlibet/quodlibet/ext/events/libraryfileorganizing.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Jonathan Schmeling
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+import os
+import re
+import shutil
+from gi.repository import Gtk
+from quodlibet import _
+from quodlibet import app
+from quodlibet import util
+from quodlibet import qltk
+from quodlibet.qltk.entry import UndoEntry
+from quodlibet.pattern import Pattern
+from quodlibet.plugins import PluginConfigMixin
+from quodlibet.plugins.events import EventPlugin
+from quodlibet.util.library import get_scan_dirs
+from quodlibet.util.library import scan_library
+
+
+class MyPlugin(EventPlugin, PluginConfigMixin):
+    PLUGIN_ID = "libraryfileorganizing"
+    PLUGIN_NAME = _("Automatically Update Library Files")
```

Comment:
@frestr I agree. When I saw the pull request, I thought this would change something inside the files (like updating the tags). That said, I think "automatically" (or similar) should still be part of the name, to make clear that this plugin doesn't require manual triggering. Maybe it could be shortened to "Auto-Update File Locations"